content/en/docs/concepts/storage/volumes.md

```diff
@@ -985,6 +985,59 @@ spec:
         claimName: my-lamp-site-data
 ```
 
+## Using subPath with expanded environment variables
+
+The following Pod definition shows how to configure unique subpaths and allow expansion of a variable host path.
```

Comment:
I think we should avoid using "host path" to not confuse it with hostPath volumes.  Maybe something like this?

"subPath directory names can also be constructed from Downward API environment variables.  Here is an example of a Pod that uses subPath to create a logs directory with the pod name."